ci-operator/jobs/openshift-ossm/maistra/openshift-ossm-maistra-postsubmits.yaml

```diff
@@ -0,0 +1,545 @@
+postsubmits:
```

Comment:
`ci-operator` has a way to build and push your images. You should add them in your ci-operator configuration and let the generated postsubmit only. Try to check https://docs.ci.openshift.org/docs/architecture/ci-operator/ to get more context about how ci-operator works.

Reply:
ok, I will update my PR and thanks for your suggestion.

Reply:
Hi @droslean , I updated my PR using the ci-operator. Could you have a review ?
Is there a doc or explanation about how the ci-operator watches a github repo and branch ?
Thanks,

Reply:
Sure,
For example, if I need to set up a presubmit test for
https://github.com/maistra/istio
branch maistra-2.2

I am adding the following in the ci-operator config:
https://github.com/openshift/release/blob/3f92cabcd546fd1f858699cf59da80aa84456020/ci-operator/config/maistra/istio/maistra-istio-2.2.yaml#L231
Is that correct ? I assume the ci-operator job will clone the github maistra/istio repo and checkout the branch.